app/Http/Controllers/V2/Dashboard/ViewProjectController.php

```diff
@@ -19,11 +19,8 @@ public function getIfUserIsAllowedToProject(String $uuid)
         /** @var User $user */
         $user = Auth::user();
         if ($user->hasRole('government')) {
-            $isAllowed = Project::where('uuid', $uuid)
-                ->where('country', $user->country)
-                ->exists();
             $response = (object)[
-                'allowed' => $isAllowed,
+              'allowed' => false,
```

Comment:
is the endpoint still needed now?

Reply:
yes, but for government now the request is that it cannot see any polygons, that is why is always false. 
Or should I check from FE ?